__tests__/params.spec.ts

```diff
@@ -2,27 +2,79 @@ import { mount } from '@vue/test-utils'
 import { watch } from 'vue'
 import { getRouter } from '../src'
 
-describe('setParams', () => {
-  it('sets current route params', async () => {
-    const router = getRouter()
-    router.setParams({ userId: 12 })
-    const wrapper = mount({
-      template: `<p>{{ $route.params.userId }}</p>`,
+describe('Params utils', () => {
```

Comment:
Let's also rename the file to partialLocation instead of params


```suggestion
describe('partial location', () => {
```